src/controller/app_routes.rs

```diff
@@ -177,49 +177,69 @@ impl AppRoutes {
         app = Self::add_powered_by_header(app, &ctx.config.server);
 
         if let Some(catch_panic) = &ctx.config.server.middlewares.catch_panic {
-            if catch_panic.enable {
+            if catch_panic.is_enabled() {
                 app = Self::add_catch_panic(app);
             }
         }
 
         if let Some(compression) = &ctx.config.server.middlewares.compression {
-            if compression.enable {
+            if compression.is_enabled() {
                 app = Self::add_compression_middleware(app);
             }
         }
 
-        if let Some(limit) = &ctx.config.server.middlewares.limit_payload {
-            if limit.enable {
-                app = Self::add_limit_payload_middleware(app, limit)?;
-            }
+        if let Some(limit) = ctx
+            .config
+            .server
+            .middlewares
+            .limit_payload
+            .clone()
+            .and_then(|l| l.into_inner())
+        {
+            app = Self::add_limit_payload_middleware(app, &limit)?;
         }
 
         if let Some(logger) = &ctx.config.server.middlewares.logger {
-            if logger.enable {
+            if logger.is_enabled() {
```

Comment:
i'm wondering why the structure here is different from the other middleware powered by the new Enable library

Reply:
Excellent catch. I've tidied my approach a buch here in a recent commit.